integration/Makefile

```diff
@@ -140,5 +140,5 @@ test:
 	set -o pipefail; \
 		CONNECT_VERSION=${CONNECT_VERSION} \
 		CONNECT_API_KEY="$(shell $(UV) run rsconnect bootstrap -i -s http://connect:3939 --raw)" \
-		$(UV) run pytest -s --junit-xml=./reports/$(CONNECT_VERSION).xml | \
+		$(UV) run pytest -s -k TestPackages --junit-xml=./reports/$(CONNECT_VERSION).xml | \
```

Comment:
```suggestion
		$(UV) run pytest -s --junit-xml=./reports/$(CONNECT_VERSION).xml | \
```

Reply:
Whoops. Good catch!

Reply:
Added a change to alleviate the need to manually changing this command as part of the development workflow https://github.com/posit-dev/posit-sdk-py/pull/358